doc/api/path.md

```diff
@@ -317,17 +317,29 @@ added: v0.11.2
 * `path` {string}
 * Returns: {boolean}
 
-The `path.isAbsolute()` method determines if `path` is an absolute path.
+The `path.isAbsolute()` method determines if the literal `path` is absolute.
+Therefore, it’s not safe for mitigating path traversals without normalizing it.
+
+```js
+// Normalizing the subpath mitigates traversing above the mount directory
+const subpath = '/foo/../..'
+if (!path.isAbsolute(path.normalize(subpath))) {
+  throw 'FORBIDDEN'
+}
+const myPath = path.join(MOUNT_DIR, subpath)
```

Comment:
This example is showing what NOT to do. E.g., if you take the following

```js
const MOUNT_DIR = "/home/example/";
const userProvidedPath = "/foo/../../../etc/passwd";

console.log(path.isAbsolute(path.normalize(userProvidedPath))); // true, i.e. it won't trigger the 'FORBIDDEN' error
console.log(path.join(MOUNT_DIR, userProvidedPath)); // /etc/passwd
```
Why not use the one I suggested? Explain to me what you don't like about it.